src/adapter/src/coord.rs

```diff
@@ -1675,8 +1675,8 @@ impl Coordinator {
 
                     // MIR ⇒ LIR lowering and LIR ⇒ LIR optimization (global)
                     let global_lir_plan = optimizer.optimize(global_mir_plan)?;
-                    let physical_plan = global_lir_plan.df_desc().clone();
-                    let metainfo = global_lir_plan.df_meta().clone();
+
+                    let (physical_plan, metainfo) = global_lir_plan.unapply();
```

Comment:
nit: Naming conventions everywhere else in the code currently use `let (df_desc, df_meta)`, maybe we should stick with that.

Reply:
I did it like this because we also have the `optimized_plan` here, which is also a `DataflowDescription`, so `df_desc` would be a bit ambiguous. The only thing we do with both of these plans is pass them to `set_optimized_plan`/`set_physical_plan`, and the variable names follow from that.